lib/resolvers/project-version.js

```diff
@@ -0,0 +1,12 @@
+const resolver = require('./base-resolver');
+
+module.exports = ({ models }) => ({ action, data, id }, transaction) => {
+  return resolver({ Model: models.ProjectVersion, action, data, id }, transaction)
+    .then(version => {
+      if (action === 'update') {
+        return models.Project.query(transaction).patchAndFetchById(version.projectId, { title: version.data.title })
```

Comment:
We might need a little more logic here at some point to determine if the parent licence is active or not. Modifying a version of an ungranted licence should update the parent, but for a granted licence it should probably only be updated when an amendment is granted.

I'm happy to let this ride for now though as we're focusing on first-time applications, and deal with the amendment case later.

Reply:
yeah we absolutely will need more logic, I added this file for the logic to live rather than just autoresolving all the things - I figured it would be better to work out what that logic is at a different point though